spec/factories/reservations.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :reservation do
+    association :user
+    association :book
+
+    returned_on {nil}
```

Comment:
**[Minor]** This is probably the same as not declaring `returned_on` here at all. 😄

Reply:
updated!